deploy/docker/Dockerfile

```diff
@@ -1,5 +1,5 @@
 # Docker image for the primary terria map application server
-FROM node:16
+FROM node:16-slim
 
 RUN mkdir -p /usr/src/app && mkdir -p /etc/config/client
 WORKDIR /usr/src/app/component
```

Comment:
The files inside this directory are owned by uid 1001 and gid 127. Is there a reason to not use `node:node` as owner?

Reply:
No reason. I'll apply the changes you made to our other dockerfile here too.